tests/testthat/test-operations.R

```diff
@@ -47,7 +50,7 @@ test_that("col_substr, start after string end", {
   expect_equal(col_substr("red", 3, 4), "d")
   expect_equal(col_substr("red", 3, 5), "d")
   expect_equal(strip_style(col_substr("\033[31mred\033[39m", 3, 4)), "d")
-  expect_equal(strip_style(col_substr("\033[31mred\033[39m", 3, 5)), "d")  
+  expect_equal(strip_style(col_substr("\033[31mred\033[39m", 3, 5)), "d")
```

Comment:
Could you please not change the whitespace? I know that trailing whitespace is annoying, but whitespace changes are annoying, too. :)